pom.xml

```diff
@@ -514,16 +516,26 @@
             <artifactId>handy-uri-templates</artifactId>
             <version>2.1.8</version>
         </dependency>
-        <!-- enforcer RequireUpperBoundDeps -->
```

Comment:
The two old entries were actually causing upper bounds errors after the Jenkins and BOM updates.